core_lib/web_helpers/flask/require_login.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+import logging
+from flask import request
+
+from core_lib.session.security_handler import SecurityHandler
+from core_lib.web_helpers.decorators import handle_exception
+logger = logging.getLogger(__name__)
+
+
+class RequireLogin(object):
+    def __init__(self, policies: list = []):
+        self.policies = policies
+
+    def __call__(self, func, *args, **kwargs):
+        @wraps(func)
+        def __wrapper(*args, **kwargs):

```

Comment:
Thank you for the changes. 
Forgive me cause I may been wrong.
After seeing that my request to split into 2 decorator cause duplicate of test. 
And feel conflicted about it now. 

I have a question that can be tested in the Django tests. 

on the Django `require_login.py`  we have `request, *args, **kwargs`
If we won't specify the `request` parameter explicitly. 
will it be included or not?

Thank you